smu/geometry/bond_length_distribution.py

```diff
@@ -319,6 +415,58 @@ def pdf(self, length):
       return 0.0
     return result
 
+  def min(self):
+    return self._df['length'].min()
+
+  def max(self):
+    if self._right_tail_dist:
+      return np.inf
+    return self._df['length'].max() + self.bucket_size
+
+
+class Mixture:
```

Comment:
Should this inherit from LengthDistribution?

Reply:
fixed